Python.gitignore

```diff
@@ -136,3 +136,7 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# IDEs
```

Comment:
Instead of writing IDEs (like combining all of them), it would be better to mention them separately so everyone should know which IDE a configuration file/directory belongs to. It will increase the clarity for readers.

**For Example**:
`# PyCharm Configuration Files`
`.idea/`
`# VS Code Configuration Files`
`.vscode/`

I purposed this change related to PyCharm configuration directory around 6 weeks ago in my PR. https://github.com/github/gitignore/pull/3769